### DISCUSSION TITLE
¿Es recomendable eliminar las ramas que ya se hayan pulleado, y que no se estén usando más?

### DISCUSSION THREAD

**Discussion by Pelayori** [Arquisoft/faq#58] [Q&A] [answered]
Tenemos varias ramas "muertas", y si seguimos sin borrarlas tendremos un gran número de ellas.

**Comment by labra** [comment, marked as answer]
En mi opinión, no hay una norma clara y depende un poco de cómo queráis tener el repositorio. 

Si son ramas que ya se han mezclado con la rama máster, quizá no hay problema en borrarlas, pero si son ramas _muertas_ que no se van a mezclar, igual es bueno mantenerlas. Por ejemplo, si un miembro del equipo crea una rama para hacer una prueba en Python y luego ese código no se utiliza, puede ser bueno dejarla abierta para demostrar que ha realizado dicha contribución.

**Reply by Pelayori** [reply]
Sí, me refiero a ramas que ya han sido mezcladas en develop/master, por lo tanto su historial de commits ya está en el arbol principal.

**Comment by pglez82** [comment]
Un factor también a tener en cuenta es como "mezcláis" las ramas a develop/master. Según lo hagáis se puede perder ese historial de commits, que puede ser interesante mantener, en ciertos casos. Os recomiendo ver este video que lo explica muy bien:

https://www.youtube.com/watch?v=0chZFIZLR_0

**Reply by Pelayori** [reply]
Gracias por la respuesta. Para el primer entregable haciamos squash merge que perdia los commits, pero hemos abandonado esa metodología desde la última entrega.
